src/content/docs/infrastructure/host-integrations/host-integrations-list/sonarqube-monitoring-integration.mdx

```diff
@@ -21,314 +21,230 @@ After setting up our SonarQube integration, we give you a dashboard for your Son
-          - Version
-          - ServerId
-          - LogLevel
-```
-4. Add the below snippet to our  `jvm-sonarqube-compute-engine-metrics.yml` file:
-```yml
-collect:
-  - domain: SonarQube
-    event_type: JVMSampleSonarQubeComputeEngineMetrics
-    beans:
-      - query: name=ComputeEngineDatabaseConnection
-        attributes:
-          - PoolInitialSize
-          - PoolActiveConnections
-          - PoolMaxActiveConnections
-          - PoolIdleConnections
-          - PoolMaxIdleConnections
-          - PoolMinIdleConnections
-          - PoolMaxWaitMillis
-          - PoolRemoveAbandoned
-          - PoolRemoveAbandonedTimeoutSeconds
-      - query: name=ComputeEngineTasks
-        attributes:
-          - PendingCount
-          - LongestTimePending
-          - InProgressCount
-          - ErrorCount
-          - SuccessCount
-          - ProcessingTime
-          - WorkerMaxCount
-          - WorkerCount
-          - WorkerUuids
-          - EnabledWorkerUuids
-```
-</Collapser>
-
-<Collapser
-    id="step3"
-    title="Configure nri-jmx files"
-  >
-
-The below YAML snippet connects your SonarQube metrics via `CONNECTION_URL` to jconsole, a Java tool that makes those metrics readable. If you're following along with the JMX integration install, this step corresponds with the configuration step.
-
-To update:
-
-1. Go to the file path below:
-```shell
-cd /etc/newrelic-infra/integrations.d/
-```
-2. Create the `jmx-sonarqube-web-config.yml` and `jmx-sonarqube-compute-engine-config.yml` files.
-3. Add the following snippet to your `jmx-sonarqube-web-config.yml` file for the SonarQube Web server.
-
-```yml
-integrations:
-  - name: nri-jmx
-    env:
-      COLLECTION_FILES: /etc/newrelic-infra/integrations.d/jmx-sonarqube-web-config.yml
-      JMX_HOST: <YOUR_HOST>
-      JMX_PASS: admin
-      JMX_PORT: 9010
-      JMX_USER: admin
-      CONNECTION_URL: service:jmx:rmi://<YOUR_IP>:9010/jndi/rmi://<YOUR_IP>:9010/jmxrmi
-      REMOTE_MONITORING: "true"
-    interval: 15s
-    labels:
-      env: staging
-```
-4. Add the following snippet to your `jmx-sonarqube-compute-engine-config.yml` file for the SonarQube Compute Engine.
-
-```yml
-integrations:
-  - name: nri-jmx
-    env:
-      COLLECTION_FILES: /etc/newrelic-infra/integrations.d/jmx-sonarqube-compute-engine-config.yml
-      JMX_HOST: <YOUR_HOST>
-      JMX_PASS: admin
-      JMX_PORT: 9011
-      JMX_USER: admin
-      CONNECTION_URL: service:jmx:rmi://<YOUR_IP>:9011/jndi/rmi://<YOUR_IP>:9011/jmxrmi
-      REMOTE_MONITORING: "true"
-    interval: 15s
-    labels:
-      env: staging
-```
-</Collapser>
-</CollapserGroup>
-
+        
+    </Step>
+    <Step>
+## Configure NRI-Prometheus      
+
+1. Run the following command to create a NRI-Prometheus config file:
+
+  ```shell
+  touch /etc/newrelic-infra/integrations.d/nri-prometheus-config.yml
+  ```
+2. Paste the following snippet into the new config file. Be sure to update the `cluster_name` and `urls` fields:
```

Comment:
update them with what? might be obvious, but do they only need to include cluster names for something related to SQ? do they need to include all cluster names? what urls (code comments say 'sq metrics lists' but why not just put that in the content part of the step) 

also, in your code comments, we should avoid using "Desired" but i'd follow up with A* about why. i can't remember but my takeaway was "don't"